.github/workflows/rust.yml

```diff
@@ -17,11 +17,11 @@ jobs:
     - uses: actions/checkout@28c7f3d2b5162b5ddd3dfd9a45aa55eaf396478b
     - uses: actions-rs/toolchain@b223206e28798aa3c3668bdd6409258e6dc29172
       with:
-        toolchain: nightly-2020-06-02
+        toolchain: nightly-2020-07-26
```

Comment:
Nit: do we need to change this, or is there some reason to do it now?  It'd just be nice to not have to change this all the time if we've got something that works.

Reply:
I accidentally upgraded locally like a numpty. I just did it for me for the moment.